<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMMWindow</class>
 <widget class="QMainWindow" name="TMMWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeleMidia Multiplexer GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Inputs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_PlaylistItems">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Playlist Items</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_Add_Item">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_Remove_Item">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QTreeView" name="treeView_Playlist">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_ItemDescription">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Item Description</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0" colspan="4">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Audio/Videos</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QTreeView" name="treeView_AVs"/>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_Add_AV">
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_Remove_AV">
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Item Name:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Duration (seconds):</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Carousel</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Application Path:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Carousel Bitrate:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_Carousel_Bitrate"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_Carousel_Path"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_Duration"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_Item_Name"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Bitrate:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Destination Address:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Use system time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_Output_Bitrate"/>
        </item>
        <item row="3" column="1" colspan="5">
         <widget class="QLineEdit" name="lineEdit_Dest_Address"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Run"/>
   <addaction name="action_Stop"/>
  </widget>
  <action name="action_Run">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/images/run_remote.png</normaloff>:/icon/images/run_remote.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start Transmission</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icon/images/stop.png</normaloff>:/icon/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Transmission</string>
   </property>
   <property name="toolTip">
    <string>Stop Transmission</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
